### ISSUE TITLE
Adding Specific Steps on "How-To" for Integration Tests

### ISSUE THREAD

**Issue by leahhoefling** [nashville-software-school/bangazon-inc#128]
We think it'd be good to add in steps on how to set up the Integration Tests so students can be guided through the process of doing so.

**Comment by stevebrownlee** [comment]
Yes, the update I just did was a quick fix, and I knew that I'd need to flesh it out more.  I'll fix this issue before cohort 30 gets there.

**Comment by askingalot** [comment]
I wrote these instructions to supplement the chapter for C29.

https://gist.github.com/askingalot/f0d459f17f332b6f50ef331b7d3f1e68
